src/components/LastDigitStats.jsx

```diff
@@ -3,19 +3,20 @@ import { connect } from '../store/Connect';
 import '../../sass/components/_last-digits.scss';
 
 const Bar = ({ x, bar }) => (
-    <div className={`cq-bar ${bar.cName}`} style={{ height: bar.height, left : (x * 35), '--bar-gradiant': `${bar.gradiantLine}%` }}>
+    <div className={`cq-bar ${bar.cName}`} style={{ height: `${bar.height * 5}%`, left : (x * 35) }}>
+        <div className={`cq-bar-value ${bar.cName}`} style={{ opacity: `${bar.height === 0 ? 0 : 1}` }}>{`${bar.height.toFixed(1)}%`}</div>
```

Comment:
@MahboobehMohammadi It'd be better to add a class to the `className` when `bar.height` is `0` instead of using style directly in the code?